packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import { Localized } from 'fluent-react';
+import { formatCurrencyInCents } from '../../lib/formats';
+import { Plan, Profile, Customer } from '../../store/types';
+
+import circledCheckbox from './images/circled-confirm.svg';
+
+import './index.scss';
+
+type PaymentConfirmationProps = {
+  customer: Customer;
+  profile: Profile;
+  selectedPlan: Plan;
+  className?: string;
+};
+
+export const PaymentConfirmation = ({
+  customer,
+  profile,
+  selectedPlan,
+  className = 'default',
+}: PaymentConfirmationProps) => {
+  const { amount, interval } = selectedPlan;
+  const { displayName, email } = profile;
+  const { brand, last4, subscriptions } = customer;
+  const invoiceNumber = subscriptions[0].latest_invoice;
```

Comment:
If a user has multiple subscriptions, will this one always be the first in the array or should we do something like find the matching `plan_name`?

(what you have could be right, just thinking out-loud here without doing extra digging)

Reply:
based on the way it was explained to me from @jackiemunroe, it should be the first in the array. But i'd like to verify

Reply:
@meandavejustice This is currently true, but it in the future it will change if we allow multiple subscriptions. I would make a note to update this in the future to search the subscriptions for the desired plan.

Reply:
IIRC, we technically support multiple subscriptions right now. We just don't have multiple products to subscribe to. And the multiple subscriptions are all independently paid for, which is not ideal and a thing we want to work on.